src/model/key.py

```diff
@@ -10,28 +11,57 @@
 @dataclasses.dataclass
 class Key:
     key_id: str
+    fingerprint: str
     key_armor: str
     key_armor_url: str
     key_user_id: KeyUserId
     usage: Usage
 
+    @staticmethod
+    def from_key_armor(key_armor: str):
+        try:
+            parsed_key = pgpy.PGPKey.from_blob(key_armor.replace("\\n", "\n"))
+            key: pgpy.PGPKey = parsed_key[0]
+            long_key_id = list(parsed_key[1].keys())[0][0]
+        except ValueError:
+            return None
+
+        return Key(
+            key_armor=key_armor,
+            key_id=long_key_id,
+            fingerprint=key.fingerprint.replace(" ", ""),
```

Comment:
```suggestion
            key_id=long_key_id,
            fingerprint=key.fingerprint.replace(" ", ""),
            key_armor=key_armor,
```

いちいち細かいことでどうでもいいかもしれないが気になってしまった